src/main/java/com/minecolonies/coremod/util/BlockUtils.java

```diff
@@ -400,4 +400,37 @@ public static boolean isPathBlock(final Block block)
     {
         return block == Blocks.GRAVEL || block == Blocks.STONEBRICK;
     }
+
+    /**
+     * Getter for a Block from an itemStack.
+     * @param stack the incoming itemStack
+     * @return the outgoing block
+     */
+    public static Block getBlockFromItemStack(final ItemStack stack)
+    {
+        Block block = Block.getBlockFromItem(stack.getItem());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'block' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)